src/core.c

```diff
@@ -5373,3 +5758,125 @@ void UWPGestureTouch(int pointer, float x, float y, bool touch)
 }
 
 #endif // PLATFORM_UWP
+
+void SetGraphicDeviceName(const char *name)
```

Comment:
This function shouldn't be required. 

In any case, it can be used: `#define DEFAULT_GRAPHIC_DEVICE_DRM "/dev/dri/card1"`

or simply let checking `card1` and `card0` (current implementation)